Add update entity platform (#22082)
home-assistant/home-assistant.github.io: source/_integrations/update.markdown

ha_category:
  - Updates
ha_release: 2022.4
ha_quality_scale: internal
ha_domain: update
ha_codeowners:
  - '@home-assistant/core'
---

An update entity is an entity that indicates if an update is available for a
device or service. This can be any update, including update of a firmware
for a device like a light bulb or router, or software updates for things like
add-ons or containers.

The state of the update entity indicates if there is an update pending or not,
and if there is an update available, more information on that update can be
provided by an integration to the entity. For example, the version that is
available, a summary of the release notes, and even links that provide more
information on the available update.

Lastly, there are two services available for the update entity. If possible and
made available by the integration providing the update entity, triggering
the actual update from Home Assistant. The other service exposed allows for
skipping the offered update.

<div class='note'>

Update entities are here to be consumed and provided by other integrations and
are are not designed to be created manually directly.

> This page, therefore, does not provide in instruction on how to create update

Reviewer: ```suggestion
This page, therefore, does not provide instructions on how to create update
```

> entities. Please see the ["Update" category](/integrations/#updates) on the

Reviewer: ```suggestion
entities. Please see the ["Updates" category](/integrations/#updates) on the
```

> integrations page to find integration offering update capabilities.

Reviewer: ```suggestion
integrations page to find integration offering update entities.
```
unsure about this one 🤷‍♂️
